tests/core.py

```diff
@@ -391,6 +391,7 @@ def test_illegal_args(self):
         Tests that Operators reject illegal arguments
         """
         with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
```

Comment:
Curious what this one is about?

Reply:
`warnings.catch_warnings` should record warnings during code execution.  But the test was failing because it was not recording any warnings. Added the filter to fix that.